katello/katello-change-hostname

```diff
@@ -237,6 +238,12 @@ def hammer_cmd(cmd, exit_codes=[0], message=nil)
   run_cmd("hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
 end
 
+def get_default_proxy_id
+  output = hammer_cmd("--output json capsule info --name #{@old_hostname}")
```

Comment:
add `message="Couldn't find default proxy"` to the `hammer_cmd` call to mimick the old `else` path below?